applications/nrf_desktop/configuration/common/hid_report_keyboard.h

```diff
@@ -23,6 +23,9 @@ extern "C" {
 #define REPORT_SIZE_KEYBOARD_KEYS	8 /* bytes */
 #define REPORT_SIZE_KEYBOARD_LEDS	1 /* bytes */
 
+/* Boot keyboard report uses the same format as keyboard keys report. */
+#define REPORT_SIZE_KEYBOARD_BOOT	REPORT_SIZE_KEYBOARD_KEYS
```

Comment:
If both use the same size why do we need to add extra define? Cannot we use one for both cases?

Reply:
I wanted to make it more explicit. HID boot mouse report has its own definition for size too.
If the HID keyboard keys report format used by nRF Desktop changes at some point, the sizes would no longer be equal too.

Reply:
maybe add this comment here?

Reply:
I already have a comment that the same format is used (it means sizes are equal too)

Reply:
If the HID keyboard keys report format used by nRF Desktop changes at some point, the sizes would no longer be equal too.

Reply:
After reading comment as it is now I wonder why to add this in the first place. Maybe if you don't want to add extra comment then remove existing one

Reply:
> If the HID keyboard keys report format used by nRF Desktop changes at some point, the sizes would no longer be equal too.

It's defined as part of the same header file, so ensuring that these values are updated together should not be a problem. The comment is added because it's specific to nRF Desktop.

Reply:
@kapi-no, @mkapala-nordic, @alstrzebonski, what do you think ?

Reply:
current version seems fine to me.

Reply:
"Boot keyboard report **currently** uses the same format as keyboard keys report."?

Reply:
`/* nRF Desktop's keyboard keys report currently uses the same format as boot keyboard reports. */`